### ISSUE TITLE
only refresh when there is a change

### ISSUE THREAD

**Issue by UXDart** [YousefED/SyncedStore#63]
Hello! if I have the sample todo list, and I do a "todo.completed = false;" always false
if the value is already false, it triggers a change in the store.. and the page refreshes. is it possible to trigger a change only if the set value is different from the current one? of course, I can check if completed is false, but would like a way to do this automatically if possible. Thanks!

**Comment by YousefED** [comment]
Hi! This should already be happening. Do you have a codesandbox link with an example?

**Comment by UXDart** [comment]
yes, check this one https://codesandbox.io/s/sandpack-project-forked-do2rj7?file=/App.tsx
completed is always set to false / click many times on the unchecked checkbox, and in the console you will see the refreshss line in the console. TIA
